boot.php

```diff
@@ -731,25 +738,27 @@ function update_db()
 	return;
 }
 
-function run_update_function($x)
+function run_update_function($x, $post_update = true)
```

Comment:
Please change the parameter from `bool $post_update` to `string $prefix = 'update'`. The prefix will be passed during the function call.